packages/beacon-node/test/sim/withdrawal-interop.test.ts

```diff
@@ -1,388 +0,0 @@
-      controller.signal
-    );
-    afterEachCallbacks.push(() => tearDownCallBack());
-    const {genesisBlockHash, engineRpcUrl} = elClient;
-    console.log({genesisBlockHash});
-
-    //const controller = new AbortController();
-    const executionEngine = initializeExecutionEngine(
-      {mode: "http", urls: [engineRpcUrl], jwtSecretHex, retries, retryDelay},
-      {signal: controller.signal, logger: testLogger("executionEngine")}
-    );
-
-    const withdrawalsVector = [
-      {Index: 0, Validator: 65535, Recipient: "0x0000000000000000000000000000000000000000", Amount: "0"},
-      {
-        Index: 1,
-        Validator: 65536,
-        Recipient: "0x0100000000000000000000000000000000000000",
-        Amount: "04523128485832663883",
-      },
-      {
-        Index: 2,
-        Validator: 65537,
-        Recipient: "0x0200000000000000000000000000000000000000",
-        Amount: "09046256971665327767",
-      },
-      {
-        Index: 3,
-        Validator: 65538,
-        Recipient: "0x0300000000000000000000000000000000000000",
-        Amount: "13569385457497991651",
-      },
-      {
-        Index: 4,
-        Validator: 65539,
-        Recipient: "0x0400000000000000000000000000000000000000",
-        Amount: "18446744073709551615",
-      },
-      {
-        Index: 5,
-        Validator: 65540,
-        Recipient: "0x0500000000000000000000000000000000000000",
-        Amount: "02261564242916331941",
-      },
-      {
-        Index: 6,
-        Validator: 65541,
-        Recipient: "0x0600000000000000000000000000000000000000",
-        Amount: "02713877091499598330",
-      },
-      {
-        Index: 7,
-        Validator: 65542,
-        Recipient: "0x0700000000000000000000000000000000000000",
-        Amount: "03166189940082864718",
-      },
-    ];
-
-    const withdrawals = withdrawalsVector.map((testVec) => ({
-      index: testVec.Index,
-      validatorIndex: testVec.Validator,
-      address: dataToBytes(testVec.Recipient, 20),
-      amount: BigInt(testVec.Amount),
-    }));
-
-    const preparePayloadParams: PayloadAttributes = {
-      // Note: this is created with a pre-defined genesis.json
-      timestamp: 47,
-      prevRandao: dataToBytes("0xff00000000000000000000000000000000000000000000000000000000000000", 32),
-      suggestedFeeRecipient: "0xaa00000000000000000000000000000000000000",
-      withdrawals,
-    };
-    const finalizedBlockHash = "0xfe950635b1bd2a416ff6283b0bbd30176e1b1125ad06fa729da9f3f4c1c61710";
-
-    // 1. Prepare a payload
-    const payloadId = await executionEngine.notifyForkchoiceUpdate(
-      ForkName.capella,
-      genesisBlockHash,
-      //use finalizedBlockHash as safeBlockHash
-      finalizedBlockHash,
-      finalizedBlockHash,
-      preparePayloadParams
-    );
-    if (!payloadId) throw Error("InvalidPayloadId");
-
-    // 2. Get the payload
-    const payloadWithValue = await executionEngine.getPayload(ForkName.capella, payloadId);
```

Comment:
I believe it's testing the engine API from the execution engine. Which should not be in our test suite. 

The behavior passively is tested in sim tests anyway.